### ISSUE TITLE
HEVC_NOMINAL_RANGE default is wrong

### ISSUE THREAD

**Issue by e00E** [GPUOpen-LibrariesAndSDKs/AMF#277]
https://github.com/GPUOpen-LibrariesAndSDKs/AMF/blob/5359f7dba51107da19b7650e5d8a40a0f04ea6d9/amf/public/include/components/VideoEncoderHEVC.h

```c++
enum AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_ENUM
{
    AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_FULL   = 1,
    AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_STUDIO = 2
};

...

// amf_int64(AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_ENUM) default = AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_FULL
#define AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE L"HevcNominalRange"
```

The comment claims that the default value is RANGE_FULL. However when I retrieve the default like so

```c++
int64_t a{-1};
encoder->GetProperty(AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE, &a);
```

I can see that the default is 0. This is especially surprising as 0 is not a valid value for this enum.

Finally I test an output file with ffprobe when setting INPUT_COLOR_PROFILE and OUTPUT_COLOR_PROFILE to FULL_709. I see that when not setting NOMINAL_RANGE (using the default) the format is `yuv420p(tv)` which is not full range going against what the comment says. If I manually set NOMINAL_RANGE to RANGE_FULL the format is the correct yuvj420p(pc).

**Comment by rhutsAMD** [comment]
Thank you for noticing this. A fix has been made and will be available in a future driver update. We will notify you once this update is made available.

Once the driver update with the fix is released, the default will be `AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_STUDIO = 0` and you can enable the full range by setting the property to `AMF_VIDEO_ENCODER_HEVC_NOMINAL_RANGE_FULL = 1`. The property is still a boolean type but `amf_int64` will work for backwards compatibility.

**Comment by rhutsAMD** [comment]
The fix is now available in the latest public Adrenalin 22.3.2 Optional driver.
